svthalia/concrexit
# Create payment for registrations and renewals

Closes #1002

### Summary

Registrations and renewals use the create_payment function now. And this PR removes all functionality of the payments app that marks an existing payment as paid.

### How to test

1. Create a registration as user
2. Confirm the mail
3. Accept the registration
4. Mark as paid
5. Check that the user is created with the correct membership

---

1. Create a renewal as user
3. Accept the renewal
4. Mark as paid
5. Check that the user is updated with the correct membership
